interfaces/Portalicious/staticwebapp.config.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://json.schemastore.org/staticwebapp.config.json",
+  "routes": [
+    {
+      "route": "/*.js",
+      "headers": {
+        "Cache-Control": "immutable, no-transform, max-age=31536000"
+      }
+    },
+    {
+      "route": "/*.css",
+      "headers": {
+        "Cache-Control": "immutable, no-transform, max-age=31536000"
+      }
+    },
+    {
+      "route": "/svg/*.svg",
+      "headers": {
+        "Cache-Control": "must-revalidate, no-transform, max-age=2592000"
+      }
+    },
+    {
+      "route": "/assets/*",
+      "headers": {
+        "Cache-Control": "must-revalidate, no-transform, max-age=2592000"
+      }
+    },
```

Comment:
This is maybe also Ionic-specific?

Reply:
`/assets/*` seems to be an angular thing so I'd be tempted to leave it